test/functional/feature_backwards_compatibility.py

```diff
@@ -66,8 +66,6 @@ def setup_nodes(self):
     def run_test(self):
         self.generatetoaddress(self.nodes[0], COINBASE_MATURITY + 1, self.nodes[0].getnewaddress())
 
-        self.sync_blocks()
```

Comment:
commit fa6c9964860cd154cf40aff74dfa038aa6006de0 says to remove `sync_all` but you are removing `sync_blocks` as well

Reply:
Thanks, adjusted the commit subject